tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -199,7 +199,12 @@ void throw_helper(const basic_string_view<charT> fmt, const Args&... vals) {
 }
 
 template <class charT, class... Args>
-void stream_helper(const basic_string_view<charT> expect, const Args&... vals) {
+void throw_helper(const charT* fmt, const Args&... vals) {
+    throw_helper(basic_string_view<charT>(fmt), vals...);

```

Comment:
Nitpick: In new code we try to construct temporaries with braces, which avoids looking like a function call:
```suggestion
    throw_helper(basic_string_view<charT>{fmt}, vals...);
```